<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1905</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(157, 157, 157);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1864</width>
         <height>2218</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>2200</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>730</y>
             <width>411</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 italic 22pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>Our Reccomendations!!</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>1690</x>
             <y>20</y>
             <width>101</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;Britannic Bold&quot;;</string>
           </property>
           <property name="text">
            <string>Logout</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_7">
           <property name="geometry">
            <rect>
             <x>1370</x>
             <y>20</y>
             <width>291</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;Britannic Bold&quot;;</string>
           </property>
           <property name="text">
            <string>Create Blogs</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_8">
           <property name="geometry">
            <rect>
             <x>1060</x>
             <y>20</y>
             <width>291</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;Britannic Bold&quot;;</string>
           </property>
           <property name="text">
            <string>Read Blogs</string>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEdit">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>1220</y>
             <width>441</width>
             <height>251</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color:black;</string>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;When you think about the most successful club in Europe, &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;who comes to mind? Manchester United, Real Madrid and Barcelona probably &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;spring straight to mind. Only two of these clubs make the list, and they&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; still donâ€™t break into the top two.....&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_6">
           <property name="geometry">
            <rect>
             <x>540</x>
             <y>1370</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);
border:2px white;</string>
           </property>
           <property name="text">
            <string>Read more...</string>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEdit_2">
           <property name="geometry">
            <rect>
             <x>1090</x>
             <y>1220</y>
             <width>481</width>
             <height>171</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
color: black;
</string>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:12pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-weight:400;&quot;&gt;Home to hundreds of World Heritage Sites, Europe welcomes millions of tourists each year. Unsurprisingly, Europe has one of the most developed accomodations....&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_9">
           <property name="geometry">
            <rect>
             <x>1350</x>
             <y>1320</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);
border:2px white;</string>
           </property>
           <property name="text">
            <string>Read more...</string>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEdit_3">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>1910</y>
             <width>471</width>
             <height>181</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
color: black;
</string>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:12pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;In 2022 I will continue sharing the brands that stand out and need to be noted, whether for their design, history, quality, or project. But above all because they....&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_11">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>1990</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);
border:2px white;</string>
           </property>
           <property name="text">
            <string>Read more...</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>80</y>
             <width>1981</width>
             <height>621</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/91810/Pictures/Screenshots/Screenshot (304).png</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>800</y>
             <width>441</width>
             <height>421</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/91810/Pictures/Screenshots/sawants.png</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>1090</x>
             <y>800</y>
             <width>481</width>
             <height>421</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/91810/Pictures/Screenshots/travellllll.png</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>1530</y>
             <width>471</width>
             <height>381</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/91810/Pictures/Screenshots/fashionnew.png</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>780</x>
             <y>470</y>
             <width>361</width>
             <height>71</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(189, 227, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>1100</x>
             <y>1490</y>
             <width>471</width>
             <height>401</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/91810/Pictures/Screenshots/WhatsApp Image 2022-02-24 at 1.33.56 PM.jpeg</pixmap>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEdit_4">
           <property name="geometry">
            <rect>
             <x>1100</x>
             <y>1880</y>
             <width>471</width>
             <height>211</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
color: black;
</string>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:12pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Cannabis is an age-old natural remedy used by many ancient civilizations. Today, research has shown that many compounds hail from basic Marijuana, primarily CBD and THC. While the former strains are used for medical purposes, the latter strains, likeÂ white widow autoflower, are used for recreational purposes. In whatever form you want to consume this natural wonder-leaf, weâ€™d like you to learn the effects it has on your body!....&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_12">
           <property name="geometry">
            <rect>
             <x>1430</x>
             <y>2050</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);
border:2px white;</string>
           </property>
           <property name="text">
            <string>Read more...</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>171</width>
             <height>81</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/91810/Pictures/Screenshots/iconnew.png</pixmap>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
